tests/it/writer/deploy_fail/decoded.storage.txt

```diff
@@ -1,2 +1,2 @@
-  [18] → new RevertingConstructor@0xBd770416a3345F91E4B34576cb804a576fa48EB1
-    └─ ← [Revert] 42
+  [18] → new <unknown>@0xBd770416a3345F91E4B34576cb804a576fa48EB1
+    └─ ← [Revert] 0x0000000000000000000000000000000000000000000000000000000000000042
```

Comment:
hmm, should this change?

this likely indicates a current regression where we fail to record the decoded value perhaps?